src/core/fonts.js

```diff
@@ -3227,7 +3227,7 @@ var Font = (function FontClosure() {
        */
       function readCmapTable(cmap, font, isSymbolicFont, hasEncoding) {
         var segment;
-        var start = (font.start ? font.start : 0) + cmap.offset;
+        var start = (font.start ? font.start : 0) + (cmap ? cmap.offset : 0);
```

Comment:
Handle this special case of the caller -- I don't think we shall read any data below.